learn/tanssi/technical.md

```diff
@@ -1,53 +1,91 @@
-2. Interoperability: While most blockchain networks operate in isolation, making it difficult for them to communicate and share assets, Polkadot provide native interoperability through XCM (Cross Consensus Message Format), connecting multiple chains facilitating asset transfer capabilities and information exchange.
-3. Governance and Upgrades: Polkadot introduces an on-chain governance model, empowering token holders to participate in decision-making and allowing for efficient upgrades and protocol improvements.
-4. Security: In smaller blockchain networks, security can be a concern due to lower levels of mining or staking participation. Polkadot’s model of shared security guarantees the same high security level even for the smaller parachain within the ecosystem.
-5. Customizability/flexibility: Polkadot's architecture encourages the creation of sovereign blockchains with high specialization, enabling developers to design chains with specific features and functionalities optimized for the unique requeriments of the use case.
+The Tanssi network and the ContainerChains can be considered as sibling chains, meaning that there is no hierarchical dependency whatsoever. Their responsibility and how they interact with each other through the relay chain will be covered in the following sections.
 
-The Polkadot Relay Chain serves as the main network that coordinates the overall operation of the platform providing shared security, consensus, and interoperability amongst the connected parachains, to accelerate the development of the web3.
+![Sibling Chains](/images/learn/tanssi/technical/light-technical-2.png#only-light)
+![Sibling Chains](/images/learn/tanssi/technical/dark-technical-2.png#only-dark)
 
-## The technology behind
+### Relay Chain {: #relay-chain } 
 
-Polkadot -and the parachains- are built with an open-source blockchain development framework called Substrate, which is bases on Rust programming language.
+Among many other responsibilities, the relay chain validates and finalizes the blocks produced by any chain participating in the ecosystem (including the ContainerChains and the Tanssi network), storing the most recent headers for each block of each chain.
 
-Substrate makes it easy for teams to develop blockchains, providing pre built core functinalities such as consensus mechanisms, staking & governance, account and assets management, networking and many more ready to use modules and libraries.
+The Tanssi protocol leverages this feature, relying on the relay chain as a means to provide the necessary data to both, Tanssi and its ContainerChains, allowing them to collaborate and validate the correctness of the block production service.
 
-The core principles of Substrate are ease of use and performance, allowing teams to create specialized blockchains for any use case, with high level of performance, flexibility, and robustness.
\ No newline at end of file
+![Relay chain](/images/learn/tanssi/technical/technical-3.png)
```

Comment:
Actually, there are some things that need to be centered in this diagram. The text doesn't seem to be aligned in the middle in a few different places i.e., "Tannsi ContainerChain" doesn't look centered in the orange square, "Root State 0x265d1" looks like it is more to the left than centered

Reply:
I know customizing these diagrams for dark and light backgrounds is probably a hassle, but i'm not a huge fan of the dark version of this image due to the white background in the Relay Chain Block - can you possibly just remove the white background and make all of the text there pink?

Reply:
Also professional nitpicker sorry... in the Relay Chain Block box ContainerChain state root and Tanssi state root should be swapped because on the left side Tanssi is on the top

I know I just left a comment about changing the text to pink, but as I keep looking at it, I actually think like you should use the same rectangle and border on the left side, for example the teal background and text copied over to the right side and replace "Tanssi state root" maybe just alter the text so it says "Tanssi State Root" instead of "Block Root State". Same with the ContainerChain.

Also it should be "State Root" instead of "Root State" and this comment applies to all the diagrams